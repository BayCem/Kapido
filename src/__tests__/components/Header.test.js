import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Header from '../../components/headerComponent/Header';

configure({ adapter: new Adapter() });
it('renders coorectly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
