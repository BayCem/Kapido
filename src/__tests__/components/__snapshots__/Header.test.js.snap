// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders coorectly 1`] = `
<header
  className="Header"
>
  <a
    className="header-logo"
    href="#"
  >
    <Logo />
  </a>
  <nav
    className="Header-menu"
  >
    <ul
      className="Header-menu__items"
    >
      <li>
        <a
          href="#"
        >
          Satış Partneri Ol
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Kapidoo Nasıl Çalışır
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Giriş Yap
        </a>
      </li>
    </ul>
  </nav>
  <a
    className="cta"
    href="#"
  >
    <Button
      className="blue-button"
      label="Üye Ol"
    />
  </a>
</header>
`;
